DB/src/main/java/io/deephaven/db/tables/utils/TableTools.java

```diff
@@ -1644,4 +1645,48 @@ public static String nullTypeAsString(final Class<?> dataType) {
         }
         return "(" + dataType.getName() + ")" + " null";
     }
+
+    public static TableDefinition tableDef(final ColumnDefinition<?>[] cols) {
+        return new TableDefinition(cols);
+    }
+
+    private static Class<?> typeFromShortName(final String dataTypeStr) {
+        switch (dataTypeStr) {
+            case "String":
+            case "string":
+                return String.class;
+            case "BigDecimal":
+            case "bigdecimal":
+                return java.math.BigDecimal.class;
+            case "StringSet":
+            case "stringset":
+                return io.deephaven.db.tables.libs.StringSet.class;
+            default:
+                return null;
+        }
+    }
+
+    public static Class<?> typeFromName(final String dataTypeStr) {
+        final Class<?> shortNameType = typeFromShortName(dataTypeStr);
+        if (shortNameType != null) {
+            return shortNameType;
+        }
+        final Class<?> dataType;
+        try {
+            dataType = ClassUtil.lookupClass(dataTypeStr);
+        } catch (ClassNotFoundException e) {
+            throw new IllegalArgumentException("Type " + dataTypeStr + " not known", e);
+        }
+        return dataType;
+    }
+
+    public static ColumnDefinition<?> colDef(
+            final String columnName, final String dataTypeStr) {
+        return ColumnDefinition.fromGenericType(columnName, typeFromName(dataTypeStr));
+    }
+
+    public static ColumnDefinition<?> colDef(
```

Comment:
This feels somehow duplicative with io.deephaven.db.tables.ColumnDefinition#fromGenericType(java.lang.String, java.lang.Class<T>); or like we are not doing it in the same coherent way as Devin's stuff.

Reply:
Is Devin's stuff usable right now?  Last time I asked (which was Friday) the answer was no.
Strictly speaking, for this diff we only need a way to access in a not-too-convoluted way ColumnDefinition "factories" from the python implementation of consumeToTable; yesterday I was actually thinking that it would be better to do an argument like
```
[ ('Symbol', 'string'), ('Price', 'double') ]
```
instead of
```
[ colDef('Symbol', 'string'), colDef('Price', 'double') ]
```
ie, expect python tuples in the argument, and apply colDef or something like it under the hood, instead of expect the user to know column definition objects/call colDef.

I'd like to do that in the next pass tho, if we agree that's a good way.